File: src/Z.Drawing/System.Drawing.Color/System.Drawing.ColorTranslator/Color.ToOle.cs
// Description: C# Extension Methods | Enhance the .NET Framework and .NET Core with over 1000 extension methods.
// Website & Documentation: https://csharp-extension.com/
// Issues: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License (MIT): https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: https://zzzprojects.com/
// Copyright © ZZZ Projects Inc. All rights reserved.
using System;
#if !NETSTANDARD
using System.Drawing;
#endif

public static partial class Extensions
{
#if !NETSTANDARD
    /// <summary>
    ///     Translates the specified  structure to an OLE color.
    /// </summary>
    /// <param name="c">The  structure to translate.</param>
    /// <returns>The OLE color value.</returns>
    public static Int32 ToOle(this Color c)
    {
        return ColorTranslator.ToOle(c);
    }
#endif
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Drawing/System.Drawing.Color/System.Drawing.ColorTranslator/Color.ToOle.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library

Imports System.Drawing

Public Module Extensions_747

	''' <summary>
	'''     Translates the specified  structure to an OLE color.
	''' </summary>
	''' <param name="c">The  structure to translate.</param>
	''' <returns>The OLE color value.</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function ToOle(c As Color) As Int32
		Return ColorTranslator.ToOle(c)
	End Function
End Module


